docs/pages/preview/database-access.mdx

```diff
@@ -357,10 +380,10 @@ Teleport Database Access is available starting from `6.0.0-alpha.1` pre-release.
 Download the appropriate version of Teleport for your platform from the table
 below, or visit our [downloads page](https://goteleport.com/teleport/download).
 
-{!docs/pages/preview/releases-table.mdx!}
```

Comment:
remove the page and the include for tests to pass

Reply:
Should we remove all includes in all versions? In 6.0/6.1  `release-table.mdx` is imported 5 times:

- https://github.com/gravitational/teleport/blob/branch/v6/docs/pages/database-access/getting-started.mdx
- https://github.com/gravitational/teleport/blob/branch/v6/docs/pages/database-access/guides/mysql-aws.mdx
- https://github.com/gravitational/teleport/blob/branch/v6/docs/pages/database-access/guides/mysql-self-hosted.mdx
- https://github.com/gravitational/teleport/blob/branch/v6/docs/pages/database-access/guides/postgres-aws.mdx
- https://github.com/gravitational/teleport/blob/branch/v6/docs/pages/database-access/guides/postgres-self-hosted.mdx

In other branches it's also imported multiple times.